2021-09-09 13:16:11,320 INFO    MainThread:2568 [internal.py:wandb_internal():86] W&B internal server running at pid: 2568, started at: 2021-09-09 13:16:11.310202
2021-09-09 13:16:11,326 INFO    WriterThread:2568 [datastore.py:open_for_write():77] open: C:\Programming\rl_cooperation\Pygame_RL_cooperation\Algorithm\DQN_SpaceCannons_PER\wandb\offline-run-20210909_131609-3mgc4m2x\run-3mgc4m2x.wandb
2021-09-09 13:16:11,326 DEBUG   HandlerThread:2568 [handler.py:handle_request():126] handle_request: run_start
2021-09-09 13:16:11,347 DEBUG   HandlerThread:2568 [meta.py:__init__():40] meta init
2021-09-09 13:16:11,347 DEBUG   HandlerThread:2568 [meta.py:__init__():54] meta init done
2021-09-09 13:16:11,348 DEBUG   HandlerThread:2568 [meta.py:probe():211] probe
2021-09-09 13:16:11,434 DEBUG   HandlerThread:2568 [git.py:repo():33] git repository is invalid
2021-09-09 13:16:11,434 DEBUG   HandlerThread:2568 [meta.py:_save_pip():58] save pip
2021-09-09 13:16:11,435 DEBUG   HandlerThread:2568 [meta.py:_save_pip():72] save pip done
2021-09-09 13:16:11,435 DEBUG   HandlerThread:2568 [meta.py:probe():249] probe done
